docs/mllib-pmml-model-export.md

```diff
@@ -7,15 +7,15 @@ displayTitle: PMML model export - RDD-based API
 * Table of contents
 {:toc}
 
-## `spark.mllib` supported models
```

Comment:
backquotes in mds cause display problems (see http://spark.apache.org/docs/latest/mllib-pmml-model-export.html)

Reply:
nice